uvloop/loop.pyi

```diff
@@ -21,7 +21,7 @@ _T = TypeVar('_T')
 _Context = Dict[str, Any]
 _ExceptionHandler = Callable[[asyncio.AbstractEventLoop, _Context], Any]
 _SSLContext = Union[bool, None, ssl.SSLContext]
-_TransProtPair = Tuple[asyncio.transports.BaseTransport, asyncio.protocols.BaseProtocol]
+_ProtocolT = TypeVar("_ProtocolT", bound=asyncio.BaseProtocol)
```

Comment:
this follows the implementation from https://github.com/python/typeshed/pull/6937/files#diff-4886af916d5482ca50a0cfea90fe0d403cc5a98e1087598a14d12c9370cfb6b4R19